### ISSUE TITLE
Infinite recursion with localAction middlewares

### ISSUE THREAD

**Issue by darky** [tiaod/moleculer-io#12]
If use `localAction` middlewares, which throw some error, it make infinite recursion on `io.call` actions
For fix it, need to place:

```
if (action.name === 'io.call') {
  return next(ctx);
}
```

on each `localAction` middleware

**Comment by tiaod** [comment]
what is `localAction`? 🤔

**Comment by darky** [comment]
This line cause infinite recursion:
https://github.com/tiaod/moleculer-io/blob/da2a53f4a341a4463a82e11d1745e69da25c0413/src/index.js#L240

Seems `err` object is recursive and it heap all free memory with stack trace:

```
<--- Last few GCs --->

[20024:0x55f4d184bc50]    20414 ms: Mark-sweep 1353.1 (1458.4) -> 1353.1 (1442.4) MB, 42.4 / 0.0 ms  (average mu = 0.147, current mu = 0.002) last resort GC in old space requested
[20024:0x55f4d184bc50]    20486 ms: Mark-sweep 1353.1 (1442.4) -> 1353.1 (1442.4) MB, 72.3 / 0.0 ms  (average mu = 0.060, current mu = 0.001) last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x274c46d1d971 <JSObject>
    0: builtin exit frame: stringify(this=0x274c46d10941 <Object map = 0xc9455003e21>,0x0d2ae35025b1 <undefined>,0x065581cc4a69 <JSFunction replacer (sfi = 0x29c3d41d83c9)>,0x0f07f1936da9 <Object map = 0x3686281c35c1>,0x274c46d10941 <Object map = 0xc9455003e21>)

    1: stringify [0x226592c38941] [/home/darky/Desktop/Projects/backend/node_modules/fast-safe-stringify/index.js:11] [bytecode=0x4e16...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: 0x55f4d01b96f1 node::Abort() [node]
 2: 0x55f4d01bafb7 node::OnFatalError(char const*, char const*) [node]
 3: 0x55f4d0351492 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0x55f4d03516eb v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0x55f4d06e9573  [node]

```

Module `fast-safe-stringify` internally used by `winston` logger

**Comment by darky** [comment]
Create PR: https://github.com/tiaod/moleculer-io/pull/13
for fix it

**Comment by tiaod** [comment]
Thank you! :smile:

**Comment by tiaod** [comment]
Released 1.0.8
